### ISSUE TITLE
New feature: Notification after runout

### ISSUE THREAD

**Issue by trgtylcnky** [MoonshineSG/Octoprint-Filament#24]
Hi,
I think it would be nice to see a notification when filament runout is detected.

**Comment by MoonshineSG** [comment]
Yes it would be. And it will be... :) IF you have Nautilus installed, the new release (in a few days) will come with notification support. And I'll add filament notification/pause.

**Comment by trgtylcnky** [comment]
Hmm, I am actually talking about octoprint web interface not the mobile device. Since you mentioned Nautilus, I am a little confused.

**Comment by MoonshineSG** [comment]
oh.... the web interface would have a "paused" notification... (i guess)

sorry if i confused you with Nautilus...

**Comment by MoonshineSG** [comment]
also, the plugin triggers and event (on the devel branch - maybe soon to be released) that can be used by other plugins to show any kind of notifications...

**Comment by Actpohomoc** [comment]
I'm using home-assistant automation notification with telegram notify
component. It is possible now to send even picture from webcam with
notification message.
Also I've setup the automation notification for start and finish the job if
the 3d printer.

7 черв. 2017 р. 13:39 "ǝuıɥsuooɯ" <notifications@github.com> пише:
